### PR TITLE
Initial makefile and travis setup

### PR DESCRIPTION
**Pull request by mikebrow**
Adds a basic makefile and travis markup.

Start of addressing issue #2

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+git:
+  depth: 150
```
Comment (MHBauer) [inline .travis.yml]: why?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+language: go
+
+sudo: required
```
Comment (MHBauer) [inline .travis.yml]: why?

File: .travis.yml (hunk 1 above)
Comment (mikebrow) [reply .travis.yml]: habit

File: .travis.yml (hunk 2 above)
Comment (mikebrow) [reply .travis.yml]: install stuff

**Comment by mikebrow** [comment]
@MHBauer requested changes made thx!

**Pull request merged**